pkgs/development/libraries/audio/lv2/default.nix

```diff
@@ -1,18 +1,32 @@
-{ lib, stdenv, fetchurl, gtk2, libsndfile, pkg-config, python3, wafHook }:
```

Comment:
If we're dropping `gtk2` from `buildInputs`, shouldn't we also drop it here?

Also, this probably supersedes #151292 then (cc @cheriimoya).

Reply:
I'm not dropping it anymore.

Exclusion of gtk2 also makes generation of some extensions not possible.

Reply:
oh, looks like I forgot to push some changes.. I'll be back on my desktop tomorrow.

Reply:
I think at this point it is preferable to drop `gtk2`. If someone uses the `scope` example plug-in for tests like [Carla](https://github.com/falkTX/Carla/blob/b8d38e395c215eed189a4849972f90af810ef971/source/tests.old/CarlaUtils2.cpp#L88) did, they can easily replace it with another plug-in. If they really want to use that plug-in, they can build a separate derivation. I would not expect anyone to actually need that and getting rid of GTK 2 on the ISO would be nice.

Reply:
> I think at this point it is preferable to drop `gtk2`.

Exactly what i thought when opening #151292

Reply:
Amended the commit to drop it here as well.